src/components/Toggle/__tests__/toggle-snapshot-tests.jest.js

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import renderer from "react-test-renderer";
+import Toggle from "../Toggle";
+
```

Comment:
let's add here also checks for   ariaLabel,
  ariaControls,
  areLabelsHidden,
  onOverrideText,
  offOverrideText